CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 0.0.5
+- `unescape_c_str` now uses NonZero to statically exclude nuls
```

Comment:
Please also include #14 and #16 in the detailed explanations (always good to explain where perf improvements come from).